pkg/machine/wsl/machine.go

```diff
@@ -571,7 +578,10 @@ func pipeCmdPassThrough(name string, input string, arg ...string) error {
 	cmd.Stdin = strings.NewReader(input)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	return cmd.Run()
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("command %s %v failed: %w", name, arg, err)
+	}
+	return nil
```

Comment:
I realize you want to get CI un-stuck for everyone.  I counted three seemingly identical repetitions of this sequence in this file.  Would you consider a followup PR to wrap these up in a common function somewhere?

Reply:
i would agree that having context as part of the error message would be beneficial.

Reply:
putting this into a function just to deduplicate the error text? Or deduplicate much more, i.e. including the cmd.Run() part?

There is much that can be improved here but I honestly don't have the time for that currently and given I have no windows setup I rather not fiddle to much with wsl code as I cannot test locally.